__tests__/simple-send-tx.test.js

```diff
@@ -0,0 +1,125 @@
+    expect(response.body.hash).toBeTruthy();
+  });
+
+  it("should accept value as string", async () => {
+    const response = await TestUtils.request
+      .post("/wallet/simple-send-tx")
+      .send({
+        address: "WPynsVhyU6nP7RSZAkqfijEutC88KgAyFc",
+        value: "1",
+      })
+      .set({ "x-wallet-id": TestUtils.walletId });
+    expect(response.status).toBe(200);
+    expect(response.body.hash).toBeTruthy();
+  });
+
+  it("should not accept transactions without address or value", async () => {
+    let response = await TestUtils.request
+      .post("/wallet/simple-send-tx")
+      .send({
+        value: 1,
+      })
+      .set({ "x-wallet-id": TestUtils.walletId });
+    expect(response.status).toBe(400);
+    expect(response.body.succes).toBeFalsy();
+
+    response = await TestUtils.request
+      .post("/wallet/simple-send-tx")
+      .send({
+        address: "WPynsVhyU6nP7RSZAkqfijEutC88KgAyFc",
+      })
+      .set({ "x-wallet-id": TestUtils.walletId });
+    expect(response.status).toBe(400);
+    expect(response.body.succes).toBeFalsy();
+  });
+
+  it("should receive an error when trying to do concurrent transactions (lock/unlock behavior)", async () => {
+    const promise1 = TestUtils.request
+      .post("/wallet/simple-send-tx")
+      .send({
+        address: "WPynsVhyU6nP7RSZAkqfijEutC88KgAyFc",
+        value: 1,
+      })
+      .set({ "x-wallet-id": TestUtils.walletId });
+    const promise2 = TestUtils.request
+      .post("/wallet/simple-send-tx")
+      .send({
+        address: "WPynsVhyU6nP7RSZAkqfijEutC88KgAyFc",
+        value: 1,
+      })
+      .set({ "x-wallet-id": TestUtils.walletId });
+
+    const [response1, response2] = await Promise.all([promise1, promise2]);
+    expect(response1.status).toBe(200);
+    expect(response1.body.hash).toBeTruthy();
+    expect(response2.status).toBe(200);
+    expect(response2.body.success).toBeFalsy();
+  });
+
+  it("should accept a custom token transaction", async () => {
+    const response = await TestUtils.request
+      .post("/wallet/simple-send-tx")
+      .send({
+        address: "WPynsVhyU6nP7RSZAkqfijEutC88KgAyFc",
+        value: 1,
+        token: {
+          name: "stub_token",
+          uid: "09",
+          symbol: "stub_token",
+        },
+      })
+      .set({ "x-wallet-id": TestUtils.walletId });
+    expect(response.status).toBe(200);
+    expect(response.body.hash).toBeTruthy();
+  });
+
+  it("should not accept a custom token transaction without funds to cover it", async () => {
+    const response = await TestUtils.request
+      .post("/wallet/simple-send-tx")
+      .send({
+        address: "WPynsVhyU6nP7RSZAkqfijEutC88KgAyFc",
+        value: 1,
+        token: {
+          name: "stub_token_2",
+          uid: "02",
+          symbol: "stub_token_2",
+        },
+      })
+      .set({ "x-wallet-id": TestUtils.walletId });
+    expect(response.status).toBe(200);
+    expect(response.body.success).toBeFalsy();
+  });
+
```

Comment:
I feel we should test that it fails when the change address does not belong to the wallet.

Reply:
And another one with correct change address and validate the change output really went there.

Reply:
1. `fails when the change address does not belong to the wallet`
Done
2. `succeeds with correct change address`
Done
3. `validate the change output really went there`
I can only test it mocking the fullnode response, but by doing it I will be only testing a mocked response.